src/views/pages/Lists/__tests__/container.spec.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import configureStore from 'redux-mock-store'
+
+import diveTo from 'TestUtils/diveToEnzyme'
+
+import { userIdSelector } from 'Store/concepts/account/selectors'
+
+import ListsPage, { ListsPageContainer } from '../container'
+
+jest.mock('Store/concepts/lists/selectors', () => ({
+  isEmptyListsSelector: jest.fn(() => false)
+}))
+
+jest.mock('Store/concepts/account/selectors', () => ({
+  userIdSelector: jest.fn(() => null)
+}))
+
+jest.mock('Store/concepts/data/selectors', () => ({
+  loadingSelector: jest.fn()
+}))
+
+jest.mock('react-router-dom', () => ({
+  withRouter: component => component
+}))
+
+describe('ListsPage', () => {
+  const store = configureStore()()
+  const props = {
+    location: {
+      search: ''
+    }
+  }
+  const wrapper = shallow(
+    <ListsPage
+      store={store}
+      {...props}
+    />
+  )
+
+  describe('with default props', () => {
+    it('matches snapshot', () => {
+      const container = diveTo(wrapper, ListsPageContainer)
+
+      expect(container).toMatchSnapshot()
+    })
+
+    describe('after componentDidUpdate', () => {
+      it('matches snapshot', () => {
+        const container = diveTo(wrapper, ListsPageContainer)
+        container.setProps({ location: { search: '?page=3' } })
+
+        expect(container).toMatchSnapshot()
+      })
```

Comment:
do you really need to test it with snapshot? what do your componentDidUpdate does?